nautobot/dcim/forms.py

```diff
@@ -2722,16 +2796,22 @@ def __init__(self, *args, **kwargs):
         # Limit LAG choices to interfaces which belong to the parent device (or VC master)
```

Comment:
And yet here the existing functionality was VC master only... so it looks like we weren't consistent already - this may need some investigation and thought.